pkgs/top-level/all-packages.nix

```diff
@@ -1809,6 +1809,8 @@ in
 
   futhark = haskell.lib.justStaticExecutables haskellPackages.futhark;
 
+  fverb = callPackage ../applications/audio/fverb { };
```

Comment:
Is this in the right section of ``all-packages.nix``? This seems fairly high up, I know there's more ``f...`` packages farther down.